src/components/context/TodoContext.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { TodosContextType } from '../../types/TodosContext';
+import { Todo } from '../../types/Todo';
+
+function useLocalStorage<T>(key: string, startValue: T): [T, (v: T) => void] {
```

Comment:
why do we need to keep some hook inside the context file? create `hooks` folder and move it there